Orange/widgets/unsupervised/owsom.py

```diff
@@ -338,6 +339,8 @@ def set_warnings():
             cont_attrs = [var for var in attrs if var.is_continuous]
             if not cont_attrs:
                 self.Error.no_numeric_variables()
+            elif len(attrs) == 1:
+                self.Error.single_attribute()
```

Comment:
But if we follow this route, check `len(cont_attrs)` not `len(attrs)`.

Reply:
No, I prefer your approach. We are not to block people from doing silly things. :)

Reply:
OK, but you must check `cont_attrs`, not `attrs` (and change the message from `(...) single column` to `(...) single numeric variable`). It doesn't help if the data has 1000 categorical features but a single numeric.

Reply:
It was before my coffee. I was sure now I am checking for cont_attrs.